README.md

```diff
@@ -111,7 +115,23 @@ for _, conn := range subscriptions {
 }
 ```
 
-## License
+### Logging
+
+The library uses [logrus](https://github.com/sirupsen/logrus) for logging at the moment.
+In the future we might remove those logs entirely to leave logging entirely to developers
+using `graphqlws`. For the moment, you can control the logging level of `graphqlws`
```

Comment:
Using "the moment" twice close to eachother. For now?

Reply:
I'm pondering the idea of allowing to pass a light in instead of enforcing logrus at some point. Perhaps it doesn't matter enough to include the "for now" comment.

Reply:
A logger, not a light.